packages/mui-joy/src/styles/extendTheme.ts

```diff
@@ -248,6 +250,9 @@ export default function extendTheme(themeOptions?: CssVarsThemeOptions): Theme {
       neutral: {
         ...defaultColors.neutral,
         ...createDarkModeVariantVariables('neutral'),
+        plainColor: getCssVarColor('palette-neutral-300'),
+        plainHoverColor: getCssVarColor(`palette-neutral-300`),
+        plainActiveColor: getCssVarColor(`palette-neutral-300`),
```

Comment:
```suggestion
```

Not necessary to add the same tokens for hover, active.

Reply:
I had to add this there because otherwise it pulls the styles from `lightColorSystem` . When I remove those, this is the hover I get:
<img width="834" alt="Screenshot 2023-08-18 at 12 36 15" src="https://github.com/mui/material-ui/assets/37222944/002a85e9-be5c-4bdc-8030-dd9b20a78f82">